backend/main/server/controllers.go

```diff
@@ -300,10 +307,10 @@ func (a *App) createProposal(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	proposal, err := helpers.createProposal(p)
+	proposal, errResponse := helpers.createProposal(p)
 	if err != nil {
```

Comment:
This won't catch because it is checking for err instead of errResponse

Reply:
got around this by creating `nilErr` of type `errorResponse{}`, then we can do the conditional check against that.